src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionAzureKeyVaultProvider.java

```diff
@@ -124,56 +198,43 @@ public SQLServerColumnEncryptionAzureKeyVaultProvider(
     /**
      * Constructs a SQLServerColumnEncryptionAzureKeyVaultProvider with a callback function to authenticate to AAD. This
      * is used by KeyVaultClient at runtime to authenticate to Azure Key Vault.
-     * 
+     *
      * @param authenticationCallback
      *        - Callback function used for authenticating to AAD.
      * @throws SQLServerException
      *         when an error occurs
      */
+    @Deprecated
```

Comment:
Do we need to deprecate this public interface? If so, why?

I do recall the previous discussion about removing it because we thought we couldn't support it via MSAL. But since we can support it, is there a compelling reason to push applications to the new interface?

Reply:
no it doesn't anymore, I've removed that